slices/perl-modules-5.38.yaml

```diff
@@ -4,12 +4,633 @@ essential:
   - perl-modules-5.38_copyright
 
 slices:
-  modules:
+  standard:
```

Comment:
Do we need the following files?

```
-- OMITTED --                                         Package
/usr/share/perl/5.38.2/App/Cpan.pm                    perl-modules-5.38
/usr/share/perl/5.38.2/ExtUtils/MANIFEST.SKIP         perl-modules-5.38
/usr/share/perl/5.38.2/ExtUtils/typemap               perl-modules-5.38
/usr/share/perl/5.38.2/Locale/Maketext/Cookbook.pod   perl-modules-5.38
/usr/share/perl/5.38.2/Locale/Maketext/TPJ13.pod      perl-modules-5.38
/usr/share/perl/5.38.2/PerlIO.pm                      perl-modules-5.38
/usr/share/perl/5.38.2/Test.pm                        perl-modules-5.38
/usr/share/perl/5.38.2/Test2.pm                       perl-modules-5.38
/usr/share/perl/5.38.2/bytes_heavy.pl                 perl-modules-5.38
/usr/share/perl/5.38.2/charnames.pm                   perl-modules-5.38
```

Reply:
Added to the SDF. Thanks for pointing out.